config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
 
+  devise_for :users, controllers: { sessions: 'users/sessions' }
```

Comment:
In line with my earlier comment, this should be `devise_for :users` and the hash options argument should be removed so that you can remove the empty controller.